src/status_im/contexts/onboarding/create_or_sync_profile/view.cljs

```diff
@@ -126,6 +127,7 @@
 
 (defn- internal-view
   [sign-in-type]
+  (rf/dispatch [:centralized-metrics/check-modal metrics-modal/view])
```

Comment:
This call and the other one in src/status_im/contexts/profile/profiles/view.cljs should also happen inside `rn/use-mount`.